dev-python/shiboken/shiboken-9999.ebuild

```diff
@@ -1,44 +1,59 @@
-# Copyright 1999-2016 Gentoo Foundation
+# Copyright 1999-2017 Gentoo Foundation
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=6
 
-PYTHON_COMPAT=( python{2_7,3_4,3_5} )
+PYTHON_COMPAT=( python{2_7,3_4,3_5,3_6} )
 
-inherit cmake-utils python-r1 git-r3
+inherit cmake-utils llvm python-r1 git-r3
 
-DESCRIPTION="A tool for creating Python bindings for C++ libraries"
-HOMEPAGE="https://wiki.qt.io/Pyside"
+DESCRIPTION="Tool for creating Python bindings for C++ libraries"
+HOMEPAGE="https://wiki.qt.io/PySide2"
 EGIT_REPO_URI=(
-	"git://code.qt.io/pyside/${PN}.git"
-	"https://code.qt.io/git/pyside/${PN}.git"
+	"git://code.qt.io/pyside/shiboken.git"
+	"https://code.qt.io/git/pyside/shiboken.git"
 )
 
 LICENSE="LGPL-2.1"
-SLOT="0"
+SLOT="2"
 KEYWORDS=""
 IUSE="test"
-
 REQUIRED_USE="${PYTHON_REQUIRED_USE}"
 
+# Minimum Qt version required, shiboken2 requires the qAsConst() function first
+# introduced by Qt 5.7.
+#
+# For safety, shiboken2 is automatically rebuilt on all Qt bumps. Even minor Qt
+# bumps could break backward compatibility or introduce new functionality.
+# Moreover, shiboken2 appears to violate Qt's privacy encapsulation.
+QT_PV="5.7.1:5="
+
+#FIXME: Update the "clang" dependency to require some minimum and possibly
+#maximum range of CLang versions. Due to insufficient upstream documentation and
+#ongoing development, this range is currently unknown.
 RDEPEND="
 	${PYTHON_DEPS}
-	dev-libs/libxml2
-	dev-libs/libxslt
-	dev-qt/qtcore:5
-	dev-qt/qtxml:5
-	dev-qt/qtxmlpatterns:5
+	sys-devel/clang:*
```

Comment:
Exactly. From what I can tell, shiboken2 only requires the clang toolchain at compilation time:

```
$ ldd /usr/lib64/python3.4/site-packages/shiboken2.so 
	linux-vdso.so.1 (0x00007ffee56fc000)
	libpython3.4m.so.1.0 => /usr/lib64/libpython3.4m.so.1.0 (0x00007f688c186000)
	libshiboken2.cpython-34m.so.2.0 => /usr/lib64/libshiboken2.cpython-34m.so.2.0 (0x00007f688bf55000)
	libstdc++.so.6 => /usr/lib/gcc/x86_64-pc-linux-gnu/4.9.4/libstdc++.so.6 (0x00007f688bbfe000)
	libgcc_s.so.1 => /usr/lib/gcc/x86_64-pc-linux-gnu/4.9.4/libgcc_s.so.1 (0x00007f688b9e7000)
	libc.so.6 => /lib64/libc.so.6 (0x00007f688b650000)
	libpthread.so.0 => /lib64/libpthread.so.0 (0x00007f688b434000)
	libdl.so.2 => /lib64/libdl.so.2 (0x00007f688b230000)
	libutil.so.1 => /lib64/libutil.so.1 (0x00007f688b02c000)
	libm.so.6 => /lib64/libm.so.6 (0x00007f688ad31000)
	/lib64/ld-linux-x86-64.so.2 (0x0000559ed7a0e000)
```

I'm probably missing something. It is a bit eerie that shiboken2 doesn't link against clang, isn't it?